### ISSUE TITLE
Subpar performance

### ISSUE THREAD

**Issue by Flohhhhh** [Flohhhhh/real-brake-lights#7]
![image](https://github.com/user-attachments/assets/0dc466d5-b4f9-4176-91a1-2c5aa00d1c26)

**Comment by bluelineparabellum** [comment]
Yeah I am still getting this issue.. I believe with more users on its worse of course.
